TauAnalysis/MCEmbeddingTools/python/customisers.py

```diff
@@ -443,9 +442,8 @@ def customisoptions(process):
         process.options = cms.untracked.PSet()
     process.options.emptyRunLumiMode = cms.untracked.string('doNotHandleEmptyRunsAndLumis')
     if not hasattr(process, "bunchSpacingProducer"):
-        process.bunchSpacingProducer = cms.EDProducer("BunchSpacingProducer")
-    process.bunchSpacingProducer.bunchSpacingOverride = cms.uint32(25)
-    process.bunchSpacingProducer.overrideBunchSpacing = cms.bool(True)
+        process.load('RecoLuminosity.LumiProducer.bunchSpacingProducer_cfi')
```

Comment:
Here as well the same question as above. Also, I wonder if it would be better to put the import in the beginning of the file, like
```python
from RecoLuminosity.LumiProducer.bunchSpacingProducer_cfi import bunchSpacingProducer as _bunchSpacingProducer
```